Resources/Lang/en_US.json

```diff
@@ -200,6 +200,7 @@
   "Investigator": "Investigator",
   "Guardian": "Guardian",
   "Addict": "Addict",
+  "Mole": "Mole",
```

Comment:
It looks like there is a Mole code here that has already been done in #275

Reply:
Yeah, the base of the branch was the mole branch.
I'll fix any conflicts that may arrise after the mole PR has been merged